src/js/components/Carousel/Carousel.js

```diff
@@ -12,199 +12,169 @@ import { Keyboard } from '../Keyboard';
 import { Stack } from '../Stack';
 import { withFocus } from '../hocs';
 
-class Carousel extends Component {
-  constructor(p) {
-    super(p);
-    this.state = { activeIndex: p.initialChild };
-  }
+const Carousel = props => {
+  const { initialChild, play: playProps } = props;
```

Comment:
can we destructure the props directly in the declaration? would be more in accordance with other components :)

Reply:
okay will do